features/upgrade/storage/upgrade.feature

```diff
@@ -344,3 +344,222 @@ Feature: Storage upgrade tests
       | ls | /mnt/hostpath |
     Then the output should contain:
       | test-before-upgrade |
+
+  # @author ropatil@redhat.com
+  # Description:
+  # Check csi snapshot controller status 
+  # Create project, pvc, deployment, file(test-before-upgrade) in deployment
+  # Create Volumesnapshot Class, Volumesnapshot
+  # Create Restore pvc/dep
+  # Check file(test-before-upgrade) in dep
+  @upgrade-prepare
```

Comment:
Do you mind removing these comments from the scenario header? Comments could be added right before each step.